Resources/doc/events.md

```diff
@@ -8,14 +8,14 @@ FSiAdminSecurityBundle provides some useful events which cause some default acti
 user instance. Its build-in subscribers:
 
 - [encode the new password](EventListener/UserEncodepasswordListener.php)
-- [ensure user is no longer enforced to change password if it was before](EventListener/UserPasswordChangedListener.php)
+- [ensure user is no longer enforced to change password if it was before](EventListener/ClearChangePasswordEnforcementListener.php)
 
 ## Password reset request event
 
 ``FSi\Bundle\AdminSecurityBundle\Event\AdminSecurityEvents::RESET_PASSWORD_REQUEST`` is fired when user requested to
 reset his/her password. Its built-in subscribers:
```

Comment:
"is fired when user requested to reset his/her password. Its built-in subscribers" => "is fired when user requests a password reset. It has following built-in subscribers"